service/src/main/java/org/databiosphere/workspacedataservice/dataimport/PfbQuartzJob.java

```diff
@@ -27,7 +43,36 @@ protected JobDao getJobDao() {
 
   @Override
   protected void executeInternal(UUID jobId, JobExecutionContext context) {
-    // TODO: AJ-1227? implement PFB import.
+    JobDataMap jobDataMap = context.getMergedJobDataMap();
+    URL url = getJobDataUrl(jobDataMap, ARG_URL);
+    Set<String> snapshotIds = new HashSet();
```

Comment:
nit: use `<>` to avoid squigglies about "Raw use of parameterized class"
```suggestion
    Set<String> snapshotIds = new HashSet<>();
```